dev/com.ibm.ws.jdbc_fat_heritage/.classpath

```diff
@@ -3,7 +3,7 @@
 	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="src" path="test-applications/heritageApp/src"/>
 	<classpathentry kind="src" path="test-applications/heritageDriver/src"/>
-	<classpathentry kind="src" path="test-bundles/jdbcHeritage/src"/>
+	<classpathentry kind="src" path="test-bundles/jdbcHeritage/resources"/>
```

Comment:
I don't think resources should be a source folder.  If the empty src location for this bundle is causing trouble, I'd say just remove the entry for it entirely.